src/app/docs/dropdown/dropdown-docs.component.html

```diff
@@ -0,0 +1,336 @@
+        </sky-dropdown-button>
+        <sky-dropdown-menu>
+          <sky-dropdown-item *ngFor="let item of items">
+            <button
+              type="button"
+              [attr.disabled]="item.disabled ? '' : null"
+              (click)="actionClicked(item.name)"
+            >
+              {{ item.name }}
+            </button>
+          </sky-dropdown-item>
+        </sky-dropdown-menu>
+      </sky-dropdown>
+    </sky-docs-demo>
+  </sky-docs-code-examples>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use dropdowns when users require access to multiple, related actions or when users require access to multiple, unrelated actions and you do not have room for multiple buttons.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-toolbar.png"
+          caption="use dropdowns to let users access multiple actions."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p>
+          Use context menu dropdowns for actions that affect individual items in a list.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-context-menu.png"
+          caption="use context menu dropdowns with items in a list."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p>
+          Use icon-only dropdowns when you do not have room for text labels. For example, use icon-only dropdowns in list toolbars when screen widths are narrow.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-icon.png"
+          caption="use icon-only dropdowns when there is not space to include a label."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don't use dropdowns as form inputs. Use HTML <sky-code>&lt;select&gt;</sky-code> fields instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-icon.png"
+          caption="use dropdowns as form inputs."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        thumbnailPath="~/assets/img/guidelines/dropdown/anatomy.png"
```

Comment:
I'm getting a broken image for Anatomy.

Reply:
Me too. Is `thumbnailPath` be `imagePath`? I think so based on digging into the HTML for the popover docs.